mapstruct/src/test/java/com/baeldung/mapper/unmappedproperties/DocumentMapperUnitTest.java

```diff
@@ -0,0 +1,45 @@
+package com.baeldung.mapper.unmappedproperties;
+
+import com.baeldung.unmappedproperties.dto.DocumentDTO;
+import com.baeldung.unmappedproperties.entity.Document;
+import com.baeldung.unmappedproperties.mapper.DocumentMapper;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.Date;
+
+import static org.junit.Assert.assertEquals;
+
+public class DocumentMapperUnitTest {
+
+    @Test
+    public void givenDocumentEntityToDocumentDto_whenMaps_thenCorrect() {
+        Document entity = new Document();
+        entity.setId(1);
+        entity.setTitle("Price 13-42");
+        entity.setText("List of positions.......");
+        entity.setModificationTime(new Date());
+
+        DocumentDTO dto = DocumentMapper.INSTANCE.documentToDocumentDTO(entity);
+
+        assertEquals(dto.getId(), entity.getId());
+        assertEquals(dto.getTitle(), entity.getTitle());
+        assertEquals(dto.getText(), entity.getText());
```

Comment:
The assertion should be `assertEquals(expected, actual)` - I think this is reversed.

Perhaps switch to `assertJ` as it's clearer `assertThat(dto.getId()).isEqualTo(entity.getId())`

Though equals == equals :) the reader who understands that JUnit assert equals goes expected first will stumble on examples like this.